SQL (#1679)
enkidevs/curriculum: sql/dql/joins/multiple-joins.md

ON pokemon_type.type_id = type.id;
```

> This is effectively join the first two tables (`pokemon_type` and `pokemon`) and then join the resulting table with the *type* table.

Reviewer: ```suggestion
This is effectively a join between the first two tables (`pokemon_type` and `pokemon`) and then a join between the resulting table and the `type` table.
```
